src/wallet/wallet.cpp

```diff
@@ -2145,11 +2145,11 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     setCoinsRet.clear();
     nValueRet = 0;
 
-    // List of values less than target
+
     pair<CAmount, pair<const CWalletTx*,unsigned int> > coinLowestLarger;
     coinLowestLarger.first = std::numeric_limits<CAmount>::max();
     coinLowestLarger.second.first = NULL;
-    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue;
+    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
```

Comment:
I think the previous comment location was correct, as it is not only about this line, but all the lines that follow (the BOOST_FOREACH block below, which populates the vector). The few lines above is also needed for that.

Adding `std::` only here is a bit pointless, the whole file uses vector, set, pair etc. unqualified. Even the `pair` on the same line is not qualified.
Maybe a separate PR could explicitly qualify all of them, if preferred.  Personally, I would like it.